packages/blade/src/components/BladeProvider/__tests__/paymentLightTheme/paymentLightTheme.native.ts

```diff
@@ -0,0 +1,1126 @@
+          active: globalColors.neutral.blueGrayLight[400],
+          disabled: globalColors.neutral.blueGrayLight[400],
+        },
+      },
+      text: {
+        primary: {
+          default: globalColors.neutral.blueGrayLight[0],
+          hover: globalColors.neutral.blueGrayLight[0],
+          focus: globalColors.neutral.blueGrayLight[0],
+          active: globalColors.neutral.blueGrayLight[0],
+          disabled: globalColors.neutral.blueGrayLight[600],
+        },
+        secondary: {
+          default: globalColors.chromatic.azure[600],
+          hover: globalColors.chromatic.azure[600],
+          focus: globalColors.chromatic.azure[600],
+          active: globalColors.chromatic.azure[600],
+          disabled: globalColors.neutral.blueGrayLight[600],
+        },
+        tertiary: {
+          default: globalColors.neutral.blueGrayLight[1000],
+          hover: globalColors.neutral.blueGrayLight[1000],
+          focus: globalColors.neutral.blueGrayLight[1000],
+          active: globalColors.neutral.blueGrayLight[1000],
+          disabled: globalColors.neutral.blueGrayLight[500],
+        },
+        link: {
+          default: globalColors.chromatic.azure[500],
+          hover: globalColors.chromatic.azure[600],
+          focus: globalColors.chromatic.azure[700],
+          active: globalColors.chromatic.azure[700],
+          disabled: globalColors.neutral.blueGrayLight[400],
+        },
+      },
+      icon: {
+        primary: {
+          default: globalColors.neutral.blueGrayLight[0],
+          hover: globalColors.neutral.blueGrayLight[0],
+          focus: globalColors.neutral.blueGrayLight[0],
+          active: globalColors.neutral.blueGrayLight[0],
+          disabled: globalColors.neutral.blueGrayLight[600],
+        },
+        secondary: {
+          default: globalColors.chromatic.azure[600],
+          hover: globalColors.chromatic.azure[600],
+          focus: globalColors.chromatic.azure[600],
+          active: globalColors.chromatic.azure[600],
+          disabled: globalColors.neutral.blueGrayLight[600],
+        },
+        tertiary: {
+          default: globalColors.neutral.blueGrayLight[1000],
+          hover: globalColors.neutral.blueGrayLight[1000],
+          focus: globalColors.neutral.blueGrayLight[1000],
+          active: globalColors.neutral.blueGrayLight[1000],
+          disabled: globalColors.neutral.blueGrayLight[500],
+        },
+        link: {
+          default: globalColors.chromatic.azure[500],
+          hover: globalColors.chromatic.azure[600],
+          focus: globalColors.chromatic.azure[700],
+          active: globalColors.chromatic.azure[700],
+          disabled: globalColors.neutral.blueGrayLight[400],
+        },
+      },
+    },
+  },
+  border,
+  motion: {
+    delay: {
+      '2xshort': 70,
+      xshort: 120,
+      short: 180,
+      long: 3000,
+      xlong: 5000,
+    },
+    duration: {
+      quick1: 70,
+      quick2: 150,
+      quick3: 200,
```

Comment:
The issue would be with `moderate`, what would we call them? `moderate` & `xmoderate`? Which one would be moderate and which one would be `xmoderate`?

Reply:
good question 🙈. from here: https://simple.wiktionary.org/wiki/moderate 

`lessModerate, moderate, moreModerate, mostModerate`

the issue is this scale of  1, 2, 3 looks very weird, and we shall try to find alternatives wherever possible

Reply:
In that case we'll have to go in this sequence
`lessModerate, moreModerate, mostModerate, moderate` or `moderate, mostModerate, moreModerate, lessModerate`

The closer you are to moderate, the more moderate you are on either the higher or lower side.

Looking at just `lessModerate` I can't tell whether its faster or slower than `moderate`

----
This might make more sense in comparison then - at least it would follow the same syntax as `2xquick` & that of `delay`,
`2xquick, xquick, quick, moderate, xmoderate, gentle, xgentle, 2xgentle`

Reply:
> Looking at just lessModerate I can't tell whether its faster or slower than moderate

this same confusion will also exist for `moderate` and `xmoderate` 😞

here's my another suggestion

```javascript
moreQuick: 70,
quick: 150,
lessQuick: 200,
moderate: 250,
lessModerate: 300,
moreGentle: 400,
gentle: 600,
lessGentle: 900,
```

Reply:
I don't mind this but the design team might want to add more values here. For example, 900 was recently added when they saw a use case. This naming convention makes it very rigid and we won't be able to extend it in any way